aiocogeo_tiler/tiler.py

```diff
@@ -57,8 +57,7 @@ def from_array(cls, array: Union[np.ndarray, np.ma.masked_array]) -> "TileRespon
         """create from numpy array"""
         if isinstance(array, np.ma.masked_array):
             return cls(data=array.data, mask=(array.mask * 255).astype("uint8"))
```

Comment:
maybe I should add `.from_masked()` method

Reply:
I think you should go one step further and add `from_array` to abstract the `if/else`.  If you have `.from_masked()` you still need the isinstance check.